components/Buttons/Button/Button.tsx

```diff
@@ -43,43 +43,34 @@ interface ButtonProps
    * Helps track in-page `event` interactions.
    */
   analyticsObject?: GoogleAnalyticsEventPropType;
+  disabled?: boolean;
```

Comment:
you can delete this and change the `Omit<ButtonHTMLAttributes<HTMLButtonElement>, 'disabled'>` to `ButtonHTMLAttributes<HTMLButtonElement>`

Reply:
Nice find. I didn't catch that.